airflow/jobs/scheduler_job.py

```diff
@@ -1018,7 +1024,7 @@ def _change_state_for_tis_without_dagrun(self,
             .filter(models.TaskInstance.state.in_(old_states)) \
             .filter(or_(
                 models.DagRun.state != State.RUNNING,
-                models.DagRun.state.is_(None)))
+                models.DagRun.state.is_(None)))  # pylint: disable=no-member
```

Comment:
What's the no member error here? This one looks unexpected to have to ignore

Reply:
The `is_` for string type of DagRun...

Reply:
+1

Reply:
Are we just ignoring the other files where we is `is_`?